src/getting-started/installation.md

```diff
@@ -1,45 +1,80 @@
 # Installation
 
-Installing ratatui is as simple as
+Installing ratatui is as simple as running the following:
+
 ```console
 cargo add ratatui
 ```
-and you can start programming TUIs.
-
-However, there are a few features that are worth a look.
-
-## Optional Features
 
-As ratatui grows and evolves, this list may change, so make sure to check the [main repo](./https://github.com/ratatui-org/ratatui) if you are unsure.
+And you can start programming TUIs.
 
-### Feature: Backend Selection
+`ratatui` is a library (i.e. a Rust crate).
+Running the above command in your console will add the latest version of `ratatui` to your project.
 
-See [Choosing a Backend](./ch01-02-choosing-a-backend.md) for more information. However, for most cases, the default `crossterm` backend is enough.
+If you are interested in adding a specific version, you can run the following:
```

Comment:
Do you think we should encourage this?

Reply:
I do because it helps set the user up with something that works more easily. Imagine as a new user, you come across beginner documentation that tells you at the very first step, "go choose a terminal backend; here are the tradeoffs between the various options". That's information overload. 

I think having that information is valuable in a separate section after we have shown how to run and set up a simple project.